src/layout/DropZone.tsx

```diff
@@ -0,0 +1,130 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import React, { useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { MdCloudUpload } from 'react-icons/md';
+
+export default function DropZone(props: {
+  color?: string;
+  children?: JSX.Element;
+}) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [alert, setAlert] = useState('');
```

Comment:
no-unused-vars errors should never be disabled. either use the variable, or remove it.

Reply:
fixed